### PR TITLE
Implement notecard to cycle through marks.

### PR DESCRIPTION
**Pull request by patmay**

### CODE REVIEW

File: test/unit/test_notecard.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    ncc.mark(1, 1)
+    assert ncc._note_card._note_state[1][1] == 0
+
+
+def test_out_of_range():
+    ncc = NCC()
+    ncc.mark(-1, 0)
+    ncc.mark(0, -1)
+    ncc.mark(6, 0)
+    ncc.mark(0, 21)
```
Comment (stephensmeal) [inline test/unit/test_notecard.py:24]: Should there be an assert statement as a result of these mark() method calls?

**Review by stephensmeal — COMMENTED** [review]
One comment on the unit test.

File: test/unit/test_notecard.py (hunk 1 above)
Comment (patmay) [reply test/unit/test_notecard.py:24]: Probably, but the point of the test is to basically not throw an out of range error.

**Review by stephensmeal — APPROVED** [review]
Looks good.

**Pull request merged**